components/threatconnect.js

```diff
@@ -77,6 +87,65 @@ polarity.export = PolarityComponent.extend({
         Ember.set(caseObj, '__severityColor', '');
     }
   },
+  parseMarkdownTables({ text, indicatorId } = {}) {
```

Comment:
For objects with key-value pairs we currently list the key-value pairs in a single table cell with just spaces between them.  I think this is visually a little difficult to parse.  In Jira we added newlines between key-value pairs.  I would suggest something similar which you can do using HTML `<br>` elements:

<img width="506" height="139" alt="image" src="https://github.com/user-attachments/assets/5db0ffb6-bcb1-4efa-bf5f-757c3e273728" />

The first row is how we currently display it.  The second row added `<br>` elements between the key-value pairs.